test/test-file-flac.ts

```diff
@@ -164,4 +164,75 @@ describe('Parse FLAC', () => {
     assert.strictEqual(common.albumartist, 'Various Artists', 'common.albumartist');
   });
 
+
+  it('Support additional FLAC comment TAG mapping "UNSYNCEDLYRICS"', async () => {
+    const filePath = path.join(
+      flacFilePath,
+      "Sister City - Swan Dive - 01 Attenborough Blues.flac"
+    );
+    const { format, common, native } = await mm.parseFile(filePath);
+
+    assert.strictEqual(format.container, "FLAC", "format.container");
+    assert.strictEqual(format.codec, "FLAC", "format.codec");
+
+    assert.strictEqual(common.lyrics instanceof Array && common.lyrics[0] && common.lyrics[0].length > 0, true);
```

Comment:
Be verbose and explicit unit test, provide clear message where and what is wrong in case of failing unit test.

```suggestion
    assert.isArray(common.lyrics, 'common.lyrics');
    assert.isNotEmpty(common.lyrics, 'common.lyrics');
```